### ISSUE TITLE
Directions for CA Rotation call out incorrect errand names

### ISSUE THREAD

**Issue by matthewfischer** [pivotal-cf/docs-pcf-security#40]
https://docs.pivotal.io/pivotalcf/2-3/security/pcf-infrastructure/api-cert-rotation.html

The directions for CA rotation call out running certain errands on service tiles: "Enable the Recreate All Service Instances errand if provided.". In practice we found that no errands had this name, but several had the errand called (something like) *Upgrade All Service Instances*. Was this the intended errand name? 

<img width="965" alt="screen shot 2019-01-15 at 4 18 57 pm" src="https://user-images.githubusercontent.com/1686293/51214099-330df280-18da-11e9-91c5-2bdc2f359aaa.png">

<img width="960" alt="screen shot 2019-01-15 at 4 19 06 pm" src="https://user-images.githubusercontent.com/1686293/51214108-36a17980-18da-11e9-8d35-7e43c999206e.png">

**Comment by ljarzynski** [comment]
Hey @matthewfischer, thanks for raising this issue! 

It looks like the line was added by @jacknewberry and @kirederik as part of: https://github.com/pivotal-cf/docs-pcf-security/pull/34. 

Jack and Derik, would you be able to speak to the issue raised by Matt ?

**Comment by kirederik** [comment]
Hey,

The errand one needs to run to rotate certs is in fact the “Recreate all” errand. 

The reason no tile currently has that errand is due to ODB didn’t make it available for the tile writers yet. We had to submit a small change to BOSH and have it backported to the version used by OpsManager, and we were blocked on releasing until then.

Fortunately, our PR was merged and backported last week, so we should be able to release it soon (and it should be integrated by the tile author short after).

Sorry for the confusion!

cc @servicesenablement

**Comment by matthewfischer** [comment]
Does this mean that cert rotation is in fact not possible until this backport lands?

**Comment by kirederik** [comment]
@matthewfischer not in an automated way.

It seems BOSH just backported to OpsManager version 2.2+ last Friday. We hope to release the errand this week. The tile teams will then integrate and release a new version of their tiles.

**Comment by ljarzynski** [comment]
@kirederik Have tile teams added this errand yet?
Also, if an ODB tile does not have this errand, are there alternative steps an operator must take as part of the CA rotation procedure? I'm wondering if we need to add anything to that page.

**Comment by jacknewberry** [comment]
Hi Landon,

I think those steps are still accurate as we wrote them. Tile teams are in
the process of adding the new errand currently to tiles for PCF 2.4 and
2.5. We are about to release a version that works for PCF 2.3 and 2.2, so
teams will be able to add the errand to those tiles next week.

Jack

**Comment by ljarzynski** [comment]
Thanks all, closing this issue as it seems to be resolved by tile changes mentioned above. 
If you find that the documentation needs to be updated, please feel free to reopen or create another issue.
